extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -510,9 +521,9 @@ export class AmpA4A extends AMP.BaseElement {
       this.experimentalNonAmpCreativeRenderMethod_ = null;
       this.rendered_ = false;
       this.timerId_ = 0;
-      if (this.apiHandler_) {
-        this.apiHandler_.unlayoutCallback();
-        this.apiHandler_ = null;
+      if (this.crossDomainIframeHandler_) {
+        this.crossDomainIframeHandler_.freeXDomainIframe();
```

Comment:
nitpick: I saw you're using XDomain and CrossDomain interchangeably,
actually, I think the formal name is cross-origin iframe.
so how about unify them to `XOrigin`.